examples/simple-kafkacluster.yaml

```diff
@@ -8,14 +8,20 @@ spec:
   zookeeperReference:
     namespace: default
     name: simple
-  opaReference:
-    namespace: default
-    name: simple-opacluster
+  opa:
+    reference:
+      namespace: default
+      name: simple-opacluster
+    authorizerClassName: com.bisnode.kafka.authorization.OpaAuthorizer
+    authorizerCacheInitialCapacity: 0
+    authorizerCacheMaximumSize: 0
+    authorizerCacheExpireAfterSeconds: 0
   brokers:
-    selectors:
+    roleGroups:
       default:
         selector:
           matchLabels:
-            kubernetes.io/hostname: debian
-        instances: 1
-        instancesPerNode: 1
\ No newline at end of file
+            kubernetes.io/hostname: mdesktop
```

Comment:
We should agree on common example hostnames to put in here at some point.

Reply:
Yeah i still sometimes forget to change to e.g. "debian" from local testing.